packages/vuepress-theme-odyssey/styles/index.scss

```diff
@@ -0,0 +1,134 @@
+@import "~@okta/odyssey";
+@import "~@okta/prism-theme-odyssey";
+
+$breakpoints: (
+  m: 768px,
+  l: 1240px,
+  lg: 1800px
+);
+
+@mixin mq($breakpoint) {
+  @if map-has-key($breakpoints, $breakpoint) {
+    $breakpoint-value: map-get($breakpoints, $breakpoint);
+
+    @media (max-width: ($breakpoint-value - 1)) {
+      @content;
+    }
+  }
+}
+
+@import "base/fonts";
+@import "base/grid";
+@import "components/OdoCard";
+@import "components/OdoLink";
+@import "components/OdoNav";
+@import "components/OdoSidebar";
+@import "templates/OdoTemplateHome";
+@import "templates/OdoTemplateComponent";
+
+$theme-day: (
+  odo-page-bg: $white,
+  odo-text-body: $text-body,
+  odo-text-heading: $text-heading,
+  odo-border-color-display: $border-color-display,
+  odo-border-color-ui: $border-color-ui,
+  odo-card-bg: cv("gray", "000"),
+  odo-card-bg-hover: cv("blue", "000")
+);
+
+$theme-dark: (
+  odo-page-bg: cv("gray", "900"),
+  odo-text-body: $white,
+  odo-text-heading: $white,
+  odo-card-bg: #303033,
+  odo-card-bg-hover: cv("gray", "600")
+);
+
+@function list-contains($list, $item) {
+  @return index($list, $item) != null;
+}
+
+@mixin setTheme($theme: $theme-day) {
+  $theme-properties: odo-page-bg, odo-text-body, odo-text-heading,
+    odo-border-color-display, odo-border-color-ui, odo-card-bg,
+    odo-card-bg-hover;
+
+  @each $custom-property, $value in $theme {
+    @if list-contains($theme-properties, $custom-property) {
+      --#{$custom-property}: #{$value};
+    } @else {
+      @warn "⚠️ Sorry, but `#{$value}` is not a valid value.";
+    }
+  }
+}
+
+html {
+  @include setTheme($theme-day);
+  background: var(--odo-page-bg);
+  color: var(--odo-text-body, #{$text-body});
+  font-family: $body-font-family;
+
+  // Theme: Night Mode
+  // &[data-odo-theme="night"] {
+  //   @include setTheme($theme-dark);
+  // }
+
+  // @media (prefers-color-scheme: dark) {
+  //   @include setTheme($theme-dark);
+  // }
+}
+
+// TODO: high-level element overrides should be in odyssey.
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  color: var(--odo-text-heading, #{$text-body});
+}
+
+p {
+  color: var(--odo-text-heading, #{$text-body});
+}
+
+a,
+a:visited {
+  &:focus {
+    @include outline;
+  }
+}
+
+.theme-odyssey {
+  display: grid;
+  grid-template-columns: 308px auto;
```

Comment:
Totally fluid option: remove `max-width` and change the grid template to `grid-template-columns: repeat(2, minmax(min-content, max-content));`

At this layer, the only known constraints could be "The site has two columns. They should always be as big as they need, but as small as they can be."

This would make the main content area completely fluid too, which will break things in the current state, but can be addressed by making similar changes in the content areas.

An interim solution would be something like `grid-template-columns: minmax(min-content, max-content) auto;` which would make the sidebar fluid, but keep the entire site "fixed" in width based on the `max-width`.